app/jobs/run_check_job.rb

```diff
@@ -1,6 +1,8 @@
 require "json/add/exception" # required to serialize errors as JSON
 
 class RunCheckJob < ApplicationJob
+  limits_concurrency to: 1, key: ->(check) { check.audit.id }
```

Comment:
Pas besoin de charger l'audit, on n'utilise que son identifiant ici.
```suggestion
  limits_concurrency to: 1, key: ->(check) { check.audit_id }
```